lint/style.py

```diff
@@ -66,7 +66,14 @@ def update(cls):
         pass
 
 
-class HighlightStyleStore(StyleBaseStore, util.Borg):
+class Borg:
+    _shared_state = {}
+
+    def __init__(self):
+        self.__dict__ = self._shared_state
+
+
+class HighlightStyleStore(StyleBaseStore, Borg):
```

Comment:
Now that's just completely insane. I do not intend to run git blame here, though.